src/WebJobs.Script/Description/CSharp/FunctionValueLoader.cs

```diff
@@ -10,7 +10,7 @@
 namespace Microsoft.Azure.WebJobs.Script.Description
 {
     /// <summary>
-    /// Lazyly loads the function value, guaranteeing single execution, and exposing 
+    /// Lazily loads the function value, guaranteeing single execution, and exposing 
```

Comment:
I discovered this and have fixed it in my pending changeset as well. Check it in :)